cmd/kubeadm/app/phases/certs/renewal/manager.go

```diff
@@ -151,6 +164,21 @@ func (rm *Manager) Certificates() []*CertificateRenewHandler {
 	return certificates
 }
 
+// CertificatesAndCertificateAuthorities return the list of certificates and certificate authorities controlled by this Manager
+func (rm *Manager) CertificatesAndCertificateAuthorities() []*CertificateRenewHandler {
```

Comment:
I think we should keep Certificates And CertificateAuthorities separated in the presentation to the users